docs/machine-learning/how-to-guides/explain-machine-learning-model-permutation-feature-importance-ml-net.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "How-To: Explain model predictions using Permutation Feature Importance"
+description: Understand the feature importance of models with Permutation Feature Importance in ML.NET
+ms.date: 04/30/2019
+author: luisquintanilla
+ms.author: luquinta
+ms.custom: mvc,how-to
+#Customer intent: As a developer, I want to use Permutation Feature Importance in ML.NET to determine the feature importance of my models so that I can understand how my machine learning models make decisions and which features contribute to their performance.
+---
+
+# How-To: Explain model predictions using Permutation Feature Importance
+
+Learn to explain ML.NET machine learning model predictions by understanding the contribution features have to predictions using Permutation Feature Importance (PFI).
+
+Machine learning models are often thought of as black boxes that take inputs and generate an output. The intermediate steps or interactions among the features that influence the output are rarely understood. As machine learning is introduced into more aspects of everyday life such as healthcare, it's of utmost importance to understand why a machine learning model makes the decisions it does. 
+
+Various techniques are used to explain models, one of which is PFI. PFI is a technique used to explain classification and regression models that is inspired by [Breiman's *Random Forests* paper](https://www.stat.berkeley.edu/~breiman/randomforest2001.pdf)(see section 10). At a high level, the way it works is by randomly shuffling data one feature at a time and calculating how much the performance metric of interest decreases. The larger the change, the more important that feature is. 
```

Comment:
>feature at a time a [](start = 341, length = 19)

I'd add: which reaks the correlation of that feature's values with the Label

Reply:
reaks?